README.md

```diff
@@ -14,15 +14,20 @@ new interface in the
 
 Each animation frame (a.k.a.
 "[rendering update](https://html.spec.whatwg.org/#update-the-rendering)")
-computes a *layout shift* (LS) score approximating the severity of visible
+computes a **layout shift (LS) score** approximating the severity of visible
 layout instability in the document during that frame.  An animation frame with
 no layout instability has an LS score of 0.  Higher LS scores correspond to
 greater instability.
 
+The LS score is based on a set of [shifting elements](#Shifting-Elements) and two
```

Comment:
Did you intend to have links here? They seem to not be working.

Reply:
They work for me at https://github.com/WICG/layout-instability/tree/0f0859440418514b540c0abe9f10d33bc3811d23 and locally with md_browser.py